backend/src/quiz/quiz.service.ts

```diff
@@ -40,7 +42,7 @@ export class QuizService {
     return this.quizModel.findAll({ where: { ownerId: userId } })
   }
 
-  async deleteOnQuizId(quizId: number, userId: number): Promise<number> {
+  async deleteOnQuizId(quizId: string, userId: number): Promise<number> {
     return this.quizModel.destroy({
       where: { id: quizId, ownerId: userId },
```

Comment:
Shouldn't this be changed to `randomId` as well?
```suggestion
      where: { randomId: quizId, ownerId: userId },
```